set.mm

```diff
@@ -556883,6 +556950,626 @@ to definitions of epsilon relation ( ~ df-eprel ) and identity relation
+          ,~ R e. Rels ) ) $=
+      ( ccoss ccnvrefrels wcel cid wss crels wa cv wbr wi wal cosselcnvrefrels2
+      wceq cossssid3 anbi1i bitri ) DEZFGUAHIZUAJGZKCLZALZDMUDBLZDMKUEUFQNBOAOC
+      OZUCKDPUBUGUCABCDRST $.
+  $}
+
+  ${
+    $d R u x $.
+    $( Necessary and sufficient condition for a coset relation to be an element
+       of the converse reflexive relation class.  (Contributed by Peter Mazsa,
+       31-Aug-2021.) $)
+    cosselcnvrefrels4 $p |- ( ,~ R e. CnvRefRels <->
+                              ( A. u E* x u R x /\ ,~ R e. Rels ) ) $=
+      ( ccoss ccnvrefrels wcel cid wss crels wa wbr cosselcnvrefrels2 cossssid4
+      cv wmo wal anbi1i bitri ) CDZEFSGHZSIFZJBNANCKAOBPZUAJCLTUBUAABCMQR $.
+  $}
+
+  ${
+    $d R x y $.
+    $( Necessary and sufficient condition for a coset relation to be an element
+       of the converse reflexive relation class.  (Contributed by Peter Mazsa,
+       5-Sep-2021.) $)
+    cosselcnvrefrels5 $p |- ( ,~ R e. CnvRefRels <->
+        ( A. x e. ran R A. y e. ran R
+          ( x = y \/ ( [ x ] `' R i^i [ y ] `' R ) = (/) ) /\
+          ,~ R e. Rels ) ) $=
+      ( ccoss ccnvrefrels wcel cid wss crels wa cv wceq ccnv cec cin c0 wo wral
+      crn cosselcnvrefrels2 cossssid5 anbi1i bitri ) CDZEFUDGHZUDIFZJAKZBKZLUGC
+      MZNUHUINOPLQBCSZRAUJRZUFJCTUEUKUFABCUAUBUC $.
+  $}
+
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  Symmetry
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+$)
+
+  $( Define the class of all symmetric sets.  It is used only by ~ df-symrels .
+
+     Note the similarity of the definitions ~ df-refs , ~ df-syms and ~~?
+     df-trs , cf. the comment of ~ dfrefrels2 .  (Contributed by Peter Mazsa,
+     19-Jul-2019.) $)
+  df-syms $a |- Syms =
+    { x | `' ( x i^i ( dom x X. ran x ) ) _S ( x i^i ( dom x X. ran x ) ) } $.
+
+  $( Define the class of all symmetric relations.  The element of the class of
+     all symmetric relations and the symmetric relation predicate are the same
+     when ` R ` is a set, cf. ~ elsymrelsrel .  Alternate definitions are
+     ~ dfsymrels2 , ~ dfsymrels3 , ~ dfsymrels4 and ~ dfsymrels5 .
+
+     This definition is similar to the definitions of the classes of all
+     reflexive ( ~ df-refrels ) and transitive ( ~~? df-trrels ) relations.
+     (Contributed by Peter Mazsa, 7-Jul-2019.) $)
+  df-symrels $a |- SymRels = ( Syms i^i Rels ) $.
+
+  $( Define the symmetric relation predicate.  (Read: ` R ` is a symmetric
+     relation.)  The element of the class of all symmetric relations and the
+     symmetric relation predicate are the same when ` R ` is a set, cf.
+     ~ elsymrelsrel .  Alternate definitions are ~ dfsymrel2 and ~ dfsymrel3 .
+     (Contributed by Peter Mazsa, 16-Jul-2021.) $)
+  df-symrel $a |- ( SymRel R <->
+      ( `' ( R i^i ( dom R X. ran R ) ) C_ ( R i^i ( dom R X. ran R ) ) /\
+        Rel R ) ) $.
```

Comment:
I believe you don't need both `Rel R` and `( R i^i ( dom R X. ran R ) )`.
Thanks tp `~ relssdmrn`, if you have the first, then the second is simply `R`.

Reply:
I see!
I suppose we usually do it the other way around, i.e. use the textbook / simplest possible statements as definition, and then prove the interesting/notable statements as theorems.

Reply:
You are right. However there is a tradeoff here: I think this is the simplest possible system of definitions with consistent numbering. I'll explain.
df-syms is the simplest definition, as I know, used only once, at least used only once by me:
df-syms $a |- Syms = { x | `' ( x i^i ( dom x X. ran x ) ) _S ( x i^i ( dom x X. ran x ) ) }
df-symrels is the simplest possible definition as well: 
df-symrels $a |- SymRels = ( Syms i^i Rels )
Here comes the numbering:
dfsymrels2 is $p |- SymRels = { r e. Rels | `' r C_ r } and
dfsymrels3 is $p |- SymRels = { r e. Rels | A. x A. y ( x r y -> y r x ) }
this logic of numbering of definitions continues long-long after symrels as well e.g.:
df-funsALTV $a |- FunsALTV = ( Funss i^i Rels )
dffunsALTV2 $p |- FunsALTV = { f e. Rels | ,~ f C_ _I }
dffunsALTV3 $p |- FunsALTV = { f e. Rels | A. u A. x A. y ( ( u f x /\ u f y ) -> x = y ) }
I should continue this df*2 df*3 logic in case of predicate forms as well:
dfsymrel2 $p |- ( SymRel R <-> ( `' R C_ R /\ Rel R ) )
dfsymrel3 $p |- ( SymRel R <-> ( A. x A. y ( x R y -> y R x ) /\ Rel R ) )
and e.g.
dffunALTV2 $p  |- ( FunALTV F <-> ( ,~ F C_ _I /\ Rel F ) )
dffunALTV3 $p |- ( FunALTV F <-> ( A. u A. x A. y ( ( u F x /\ u F y ) -> x = y ) /\ Rel F ) )
Now the question is: what should be the base definition of the predicate form _supposed that df*2 and df*3 are given_?
The answer is that it should be the predicate form of the most simple form of the class of all related specific sets ( = similar to a simplest not-predicate form). In case of functions, both form are quite simple:
df-funss $a |- Funss = { x | ,~ x e. CnvRefRels }
df-funALTV $a |- ( FunALTV F <-> ( CnvRefRel ,~ F /\ Rel F ) )
in case of symmetry, they seems to be not so simple 
df-syms $a |- Syms = { x | `' ( x i^i ( dom x X. ran x ) ) _S ( x i^i ( dom x X. ran x ) ) }
df-symrel $a |- ( SymRel R <-> ( `' ( R i^i ( dom R X. ran R ) ) C_ ( R i^i ( dom R X. ran R ) ) /\ Rel R ) )
but 1. df-syms is the simplest possible and 2. df-symrel is the simplest possible supposed that supposed that  we want to keep a consistent numbering.

Reply:
@tirix if you have an idea for a better labeling system, I'll consider to use it. But it seems to me that if we insist on consistent numbering, then either df-symrels or df-symrel can not be the simplest.